components/DoorListDock.tsx

```
@@ -0,0 +1,1412 @@
+     setHramFilter('')
+     setGuidFilter('')
+   }
+
+   const hasLocalFilters = doorFilter || typeFilterSet.size > 0 || storeyFilterSet.size > 0 || brandschutzFilterSet.size > 0 || schallschutzFilterSet.size > 0 || lbFilter || lhFilter || rbFilter || rhFilter || bramFilter || hramFilter || guidFilter
+
+   const uniqueTypeValues = useMemo(() => {
+     const s = new Set<string>()
+     doorsForTypeOptions.forEach(d => {
+       s.add(d.csetStandardCH?.geometryType || '—')
+     })
+     typeFilterSet.forEach(v => s.add(v))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)